tests/rptest/wasm/native_kafka_consumer.py

```diff
@@ -19,11 +22,11 @@ class NativeKafkaConsumer(BackgroundTask):
     def __init__(self,
                  brokers,
                  topic_partitions,
-                 num_records=1,
+                 throughput_spec,
                  batch_size=4092):
         super(NativeKafkaConsumer, self).__init__()
         self._topic_partitions = topic_partitions
-        self._num_records = num_records
+        self._throughput_spec = throughput_spec
```

Comment:
When I think of throughput, I usually think of messages per second.  From the way it's used, this variable seems like a total expected message count?  Maybe the name could be clearer.